packages/script.py

```diff
@@ -45,7 +45,7 @@ def gen_bzl_config(tag: str, dist: Path) -> None:
     packages = [package["name"] for package in lock["packages"].values()]
     imports_to_test = import_tests.gen(packages)
 
-    Path("pyodide_bucket.bzl").write_text(
+    Path(dist / "pyodide_bucket.bzl").write_text(
```

Comment:
Are we still using `pyodide_bucket.bzl` itself for anything? Maybe we could remove it?

Reply:
For now we are, yes. I'll follow up with removing it later.